DependencyInjection/MonologExtension.php

```diff
@@ -854,6 +899,7 @@ private function getHandlerClassByType($handlerType)
             'socket' => 'Monolog\Handler\SocketHandler',
             'pushover' => 'Monolog\Handler\PushoverHandler',
             'raven' => 'Monolog\Handler\RavenHandler',
+            'sentry' => 'Sentry\Monolog\Handler',
```

Comment:
Raven is the old version, it's completely incompatible with the new version (`sentry/sentry` 2.0).

Reply:
That's why I kept the `raven` type, and introduced `sentry` type for the 2.0.